docs/reference/search/vector-tile-api.asciidoc

```diff
@@ -0,0 +1,717 @@
+{wikipedia}/Well-known_text_representation_of_geometry[Well Known Text].
++
+Other field data types do not support this parameter.
+====
+
+`grid_precision`::
+(Optional, integer) Determines grid size for the `geotile_grid`. Accepts
+`0`-`8`. If `0`, results don't include the `aggs` layer. Defaults to `8`.
++
+The request calculates the grid size as:
++
+`(2^grid_precision) x (2^grid_precision)`
++
+For example, a value of `8` divides the tile into a grid of 256 x 256 cells. The
+`aggs` layer only contains features for cells with matching data.
+
+`grid_type`::
+(Optional, string) Determines the geometry of features in the `aggs` layer. In
+the `aggs` layer, each feature represents a cell of the `geotile_grid`. Accepts:
+
+`grid` (Default):::
+A polygon of the cell's bounding box.
+
+`point`:::
+A point that's the centroid of the cell.
+
+`query`::
+(Optional, object) <<query-dsl,Query DSL>> used to filter documents for the
+search.
+
+include::{es-repo-dir}/search/search.asciidoc[tag=runtime-mappings-def]
+
+`size`::
+(Optional, integer) Maximum number of features to return in the `hits` layer.
+Defaults to `1000`. If `0`, results don't include the `hits` layer.
+
+`sort`::
+(Optional, array of <<sort-search-results,sort objects>>) Sort features in the
+`hits` layer.
+
+[role="child_attributes"]
+[[vector-tile-api-response]]
+==== Response
+
+Returned vector tiles contain the following data:
+
+`hits`::
+(object) Layer containing results for the `geo_bounding_box` query.
++
+.Properties of `hits`
+[%collapsible%open]
+====
+// tag::extent[]
+`extent`::
+(integer) Size, in pixels, of a side of the vector tile. Vector tiles are square
+with equal sides.
+// end::extent[]
+
+// tag::version[]
+`version`::
+(integer) Major version number of the
+https://github.com/mapbox/vector-tile-spec[Mapbox vector tile specification].
+// end::version[]
+
+`features`::
+(array of objects) Array of features. Contains a feature for each document that
+matches the `geo_bounding_box` query.
++
+.Properties of `features` objects
+[%collapsible%open]
+=====
+// tag::geometry[]
+`geometry`::
+(object) Geometry for the feature.
++
+.Properties of `geometry`
+[%collapsible%open]
+======
+`type`::
+(string) Geometry type for the feature. Valid values are:
+
+* `UNKNOWN`
+* `POINT`
+* `LINESTRING`
+* `POLYGON`
+
+`coordinates`::
+(array of integers or array of arrays) Tile coordinates for the feature.
+======
+// end::geometry[]
+
+`properties`::
+(object) Properties for the feature.
++
+.Properties of `properties`
+[%collapsible%open]
+======
+`_id`::
+(string) Document `_id` for the feature's document.
+
+`<field>`::
+Field value. Only returned for fields in the `fields` parameter.
+======
+// tag::feature-id[]
+`id`::
+(integer) Unique ID for the feature within the layer.
+// end::feature-id[]
+
+// tag::feature-type[]
+`type`::
+(integer) Identifier for the feature's geometry type. Values are:
++
+* `1` (`POINT`)
+* `2` (`LINESTRING`)
+* `3` (`POLYGON`)
```

Comment:
This could be wrong.  I intuited this but couldn't find any reference to this field in https://github.com/mapbox/vector-tile-spec. Let me know if I'm off base here.

Reply:
That's correct :)